libs/shared/connected/src/lib/SignatureLists/SignatureLists.tsx

```diff
@@ -16,19 +19,48 @@ export const SignatureLists: FC<
   React.PropsWithChildren<SignatureListsProps>
 > = ({ slice }) => {
   const { collection, loading } = useGetCurrentCollection()
+  const { openLists } = useGetOpenLists()
   const t = useLocalization(slice.json)
 
   return (
-    !loading &&
-    collection?.candidates.length > 0 && (
+    !loading && (
       <Box marginTop={7}>
         <Box marginBottom={3}>
           <Text variant="h4">
             {t('title', 'Frambjóðendur sem hægt er að mæla með')}
           </Text>
         </Box>
         <Stack space={4}>
-          {collection?.candidates?.length > 0 &&
+          {/* if collection time is over yet there are still open lists, show them */}
+          {new Date() > new Date(collection.endTime) && openLists?.length ? (
+            openLists?.map((list: any) => {
```

Comment:
Having this typed as any is not desirable, I'd appreciate it if we could add a correct type here 🙃

Reply:
💯